packages/devtools/lib/src/timeline/cpu_profile_protocol.dart

```diff
@@ -12,13 +13,34 @@ import '../utils.dart';
 // debug and profile builds. Do they use different clocks and does this also
 // affect the sampling rate? See https://github.com/dart-lang/sdk/issues/36583.
 
+// Switch this flag to true collect debug info from the latest cpu profile. This
+// will also add a button to the timeline page that will download the debug info
```

Comment:
Nit: make this a /// comment and add a line break between the first and subsequent sentences.